modules/appeals_api/spec/serializers/evidence_submission_serializer_spec.rb

```diff
@@ -19,8 +25,24 @@
     expect(rendered_hash[:appeal_id]).to eq evidence_submission.supportable.id
   end
 
+  it 'includes :code' do
+    expect(rendered_hash[:code]).to eq evidence_submission.code
+  end
+
   it "truncates :detail value if longer than #{described_class::MAX_DETAIL_DISPLAY_LENGTH}" do
     max_length_plus_ellipses = described_class::MAX_DETAIL_DISPLAY_LENGTH + 3
     expect(rendered_hash[:detail].length).to eq(max_length_plus_ellipses)
   end
+
+  it 'includes :created_at' do
+    expect(rendered_hash[:created_at]).to eq evidence_submission.created_at
+  end
+
+  it 'includes :updated_at' do
+    expect(rendered_hash[:updated_at]).to eq evidence_submission.updated_at
+  end
+
```

Comment:
Is there any way to test the conditional nature of location rendering? What about the delegation of status to object?

Reply:
Can you please reword or explain? I don't understand.

Reply:
the `location` method in the serializer has an if statement that changes the output of the method - the delegation means that there's a dependency on another object `delegate :status to`

Reply:
I understand that.